### DISCUSSION TITLE
Authentication workflow

### DISCUSSION THREAD

**Discussion by vansu99** [NoahDuongMaster/vibe-code-stack-for-ceos#22] [Ideas]
Hi anh @truongdn-it , em nghĩ mình nên triển khai feature Authentication cho project này. Vì feature này đa số các project đều có và cũng như giúp project đa dạng hơn. Thank anh.

**Comment by NoahDuongMaster** [comment]
@vansu99 Hi bạn (chắc bằng tuổi thấy 99),

Bạn cứ triển thôi, rồi tạo PR cho mình :))
https://github.com/truongdn-it/nextjs-boilerplate/issues/23

Cái này thực ra mình làm ở mấy dự án khác rồi, định mấy nữa rảnh mình nhét vào mà mình bận quá. Bạn cứ làm đi rồi mình merge cho.
Thank!

**Reply by NoahDuongMaster** [reply]
@vansu99 Hi bạn,

https://github.com/truongdn-it/nextjs-boilerplate/issues/24

Có 2 issue cho GraphQL và ResfulAPI. Bạn thích làm cái nào cũng được.
Thank!
